cg/apps/gt.py

```diff
@@ -37,3 +46,31 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+    def get_trending(self, sample_id: str = '', days: str = '') -> dict:
```

Comment:
seems weird to have days as a str and not an int. Any reason for that?

Reply:
Nope.